docs/developer/decision-records/2022-06-13-registration-service-api/README.md

```diff
@@ -0,0 +1,49 @@
+# Registration Service API
+
+Registration Service is a component of a Dataspace Authority. 
+One of the responsibilities of the Registration Service is the management of the participants in the Dataspace.
```

Comment:
I'd say that that is it's _only_ responsibility. If not, I'd list the other responsibilities.